rest/route/github.go

```diff
@@ -600,7 +600,7 @@ func (gh *githubHookApi) tryDequeueCommitQueueItemForPR(pr *github.PullRequest)
 		return nil
 	}
 
-	_, err = gh.sc.CommitQueueRemoveItem(projRef.Identifier, strconv.Itoa(*pr.Number))
+	_, err = gh.sc.CommitQueueRemoveItem(projRef.Identifier, strconv.Itoa(*pr.Number), *pr.Base.Repo.Owner.Login)
```

Comment:
This is actually the repo owner, not the user. But this function is called when someone has closed the PR, and I don't know if that's necessarily the user that created the PR. Before this function is called we try to abort patches and it looks like we use a special user, which still may be most accurate in this case. https://github.com/evergreen-ci/evergreen/blob/9d5c9206f7ed0b12474baf2111a651ac1352044c/model/patch_lifecycle.go#L475

Reply:
interesting! thank you!